srcdoc/configure/h2olog.mt

```diff
@@ -0,0 +1,81 @@
+? my $ctx = $main::context;
+? $_mt->wrapper_file("wrapper.mt", "Configure", "Using h2olog for Tracing")->(sub {
+
+<p>h2olog is a an experimental <a href="https://www.kernel.org/doc/html/latest/bpf/index.html">BPF</a> (<a href="https://www.kernel.org/doc/Documentation/networking/filter.txt">kernel doc</a>) backed tracing tool for the <a href="https://github.com/h2o/h2o">H2O</a> server.
+It can be used for tracing HTTP-level or QUIC-level events.</p>
+
+<p><em>Since this is experimental, its command-line interface might change without notice.</em></p>
```

Comment:
In general, you should go easy on the use of "this" and "it". My suggestion is: "Since h2olog is an experimental program, its command-line interface might change without notice". I'm able to use "it" in my suggestion because the subject was clarified at the beginning of the sentence.

Reply:
It makes really sense! Fixed 670449931e4382958bacc5368f94907d0eb7b7d0